### PR TITLE
Deletion of posts

### PR DESCRIPTION
**Pull request by Glombort**
Can now delete posts. Got rid of the bit that was copied over to the home.js file as well.

Also did some tests but haven't added these as wasn't sure on how to go about a deletion test without altering the page for other tests.

### CODE REVIEW

File: routes/home.js [hunk 1]
```diff
@@ -7,9 +7,20 @@ server.use(staticHandler);
...
+    post = blogPosts[postKey]
     posts += `
-        <div class="blog-post"><div><h3>${post.user}</h3> <p>${post.message}</p></div><button class="delete">delete</button></div>
+        <div class="blog-post">
+          <div>
+            <h3>${post.user}</h3>
+            <p>${post.message}</p>
+          </div>
+          <form action="/delete-post" method="POST">
+            <button class="delete" name="name" value="${postKey}" aria-label="Delete ${postKey}">
```
Comment (mariaalouisaa) [inline routes/home.js:19]: `value="${postKey}" ` thats super clever & nice one on adding the aria-label too

File: server.js [hunk 2]
```diff
@@ -1,34 +1,18 @@
 const express = require("express");
 const home = require("./routes/home.js");
+const deletePost = require("./routes/deletePost.js")
 
 const server = express();
 
 const staticHandler = express.static("public");
 server.use(staticHandler);
 
+const bodyParser = express.urlencoded({ extended: false });
```
Comment (mariaalouisaa) [inline server.js:10]: I still need to wrap my head around this, but have faith you know what you're doing!

**Review by mariaalouisaa — COMMENTED** [review]
I happy for this to be merged, looks great!

**Review by AsmahanM — APPROVED** [review]

**Comment by AsmahanM** [comment]
Amazing, thank you!

**Pull request merged**